app/models/task.rb

```diff
@@ -269,16 +269,54 @@ def trigger_start
     return 0 if self.root?
 
     case
-    when on_started?
-      return self.parent.trigger_start
-    when on_done?
-      return (self.parent.trigger_start + self.parent.duration)
-    else
-      return "case not coded yet"
+      when on_started?
```

Comment:
Point taken, but we've been pretty consistently doing it this way for this project. I think if we want to switch to the conventional way, it may be easier to have a separate issue for it and do a dedicated find and replace of all instances in one go.